src/v/config/configuration.cc

```diff
@@ -72,7 +72,7 @@ configuration::configuration()
       "coproc_max_inflight_bytes",
       "Maximum amountt of inflight bytes when sending data to wasm engine",
       required::no,
-      128_KiB)
+      10_MiB)
```

Comment:
This is more than an or of magnitude larger than the guidance in the commit message. how should we think about tuning value when writing documentation, for example?

Reply:
The main difference is that this value represents a resource per shard, shared across all script fibers, whereas the other config values are local to a script fiber